src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -107,66 +110,64 @@ const errorDetailsMap = {
       .setStat({
         [TAG_NAMES.ERROR_TYPE]: ERROR_TYPES.AUTH,
       })
-      .setMessage(
-        'The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons',
-      )
+      .setMessageField('error_user_msg')
       .build(),
 
     463: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
       .setStat({
         [TAG_NAMES.ERROR_TYPE]: ERROR_TYPES.AUTH,
       })
-      .setMessageField('message')
+      .setMessageField('error_user_msg')
       .build(),
     default: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
       .setStat({
         [TAG_NAMES.ERROR_TYPE]: ERROR_TYPES.AUTH,
       })
-      .setMessage('Invalid OAuth 2.0 access token')
+      .setMessageField('error_user_msg')
```

Comment:
make sure `error_user_msg` indeed has more information that the manual error messages configured before

Reply:
I don't see much changes in the network mocks with `error_user_msg` scenario

Reply:
> make sure error_user_msg indeed has more information that the manual error messages configured before

Yes. We have an example for the `Invalid Parameter` error, where in `error_user_msg` contains an error message to be shown for the user

> I don't see much changes in the network mocks with error_user_msg scenarios

Copied the message into `error_user_msg` field as didn't know what would the response be actually.
https://github.com/rudderlabs/rudder-transformer/pull/3908/files#diff-371cb884d55a278880bed7da54e3c846e7d83e4d0c07ad5d14d5fa6e25e5afb4R62